### DISCUSSION TITLE
chatgpt新的会话，为什么中文回答得好好的，突然就用英文来回答我的问题？

### DISCUSSION THREAD

**Discussion by kof8855** [lobehub/lobehub#2900] [LLM Usage]
经常遇到这样的情况，新建了一个会话，问题都是持续的，本来chatgpt用中文回复得好好的，持续了十几条，突然最新的提问就用英语来回答我，而且回答的内容也是能关联上的。
这是什么问题？是我的api贩子平台的问题吗？还是lobe-chat的问题？
最近经常会这样，而且新建会话用中文提问，偶尔也会先用英文来回答。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
大概率是你服务商的问题，我自己没遇到过

[a comment by an automated account is not shown]

**Comment by Sun-drenched** [comment]
可能性很多，也许和你选用的大模型本身有关，我有时在OpenAI官网使用4o模型也会出现你说的新建会话用中文提问，偶尔也会先用英文来回答的情况，特别是提问参杂英文时；在LobeChat上我也没有遇到过这种情况，原理上来讲，LobeChat也不应该会导致这个问题。你可以试试其他二道贩子的API或者换个大模型

[a comment by an automated account is not shown]
